common/beerocks/bwl/include/bwl/base_wlan_hal_types.h

```diff
@@ -127,9 +127,11 @@ typedef struct {
 //ap_wlan_hal
 
 enum eWiFiBandwidth : uint8_t {
```

Comment:
Why don't we use the `eWiFiBandwidth` from `beerocks_defines`?

Reply:
Good question! And why don't we use the struct from tlvf? They are the same, and we have a duplication for every message.
The reason is that we didn't want to create dependency (or dependency of messages and types, not sure) from other libs such BCL.
This is really stupid IMO, since we already depended on BCL for some utilities.